src/quarot/rtn_utils.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+# Based on the implementation in https://github.com/spcl/QuaRot/
+
+import torch
+import tqdm
+
+from slicegpt import utils
+
+from .quant_utils import WeightQuantizer, find_qlayers
+
+torch.backends.cuda.matmul.allow_tf32 = False
+torch.backends.cudnn.allow_tf32 = False
```

Comment:
These 2 lines introduce unexpected side effects into any code that imports (possibly indirectly) this file.

Reply:
@sashkboos are these necessary in general or only for rtn/gptq?

Reply:
@nailimixaM 
Yes, those are used for the numerical stability of simulations. Actually, they disable using TensorCores. 


@msdmkats  
Can you please let me know what kind of error you get? maybe you are using the code with non-NVIDIA hardware?


BTW, you can remove those lines and try to compare the results. Maybe the differences is small.